workers/event_catcher/event_catcher.rb

```diff
@@ -18,17 +18,18 @@ def run!
 
     notify_started
 
-    logger.info("Collecting events...")
+    log_prefix = "MIQ(Vmware::InfraManager::EventCatcher)".freeze
```

Comment:
Super-minor but should this be `"ManageIQ::Providers::Vmware::InfraManager::EventCatcher"` ? I know it's a mouthful, but wasn't sure if that would keep it consistent or not?

Reply:
Yeah just a copy&paste error